feed/templates/feed/feed.html

```diff
@@ -1,4 +1,4 @@
-{% extends "feed/base_template.html" %}
+{% extends "jobseeker/base_template.html" %}
```

Comment:
It is best to remove the whole `feed` folder entirely (including this file). This PR doesn't require the `feed` folder right?

Reply:
You're right, I'll do it